src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PENamespaceSymbol.cs

```diff
@@ -47,6 +47,11 @@ internal abstract class PENamespaceSymbol
         /// </summary>
         private ImmutableArray<PENamedTypeSymbol> _lazyFlattenedTypes;
 
+        /// <summary>
+        /// All namespace and type members in a flat array
+        /// </summary>
+        private ImmutableArray<NamespaceOrTypeSymbol> _lazyFlattenedNamespacesAndTypes;
```

Comment:
any sorting on this?  like are all the namespaces beffore the types?  (might be useful for grabbing out subportions).

Reply:
It's not explicitly sorted, but the way things are inserted in the code means that currently all namespaces precede all types.

Reply:
changed to have types before namespaces to match old behavior